modules/installation-configuration-parameters.adoc

```diff
@@ -1166,6 +1203,61 @@ Additional IBM Cloud VPC configuration parameters are described in the following
 --
 endif::ibm-cloud[]
 
+ifdef::ibm-power-vs[]
+[id="installation-configuration-parameters-additional-ibm-cloud_{context}"]
+== Additional IBM Power VS configuration parameters
+
+Additional IBM Power VS configuration parameters are described in the following table:
+
+.Additional IBM Power VS parameters
+[cols=".^1,.^6a,.^3a",options="header"]
+|====
+|Parameter|Description|Values
+
+|`platform.powervs.resourceGroupName`
+|The name of an existing resource group to install your cluster to. This resource group must only be used for this specific cluster because the cluster components assume ownership of all of the resources in the resource group.
+|String, for example `existing_resource_group`.
```

Comment:
String. For example: ....

Also change further down in the table.

Reply:
@lpettyjo I've seen you've taken the peer-review 
Alisha and me just noticed there is some inconsistency in the file:  
The Azure part (line 1038 , 1594 etc.) uses: `String, for example ` 
Further down starting at line 1183 it always says:  `String. For example:`
I would vote for the latter.
cc @mjpytlak

Reply:
> The Azure part (line 1038 , 1594 etc.) uses: `String, for example `
> Further down starting at line 1183 it always says: `String. For example:`
> I would vote for the latter.

I agree.